main/src/main/scala/sbt/Defaults.scala

```diff
@@ -898,7 +898,7 @@ object Defaults extends BuildCommon {
         scalaRun.value.run(mainClass, data(classpath.value), parser.parsed, streams.value.log).get
       }
     }
-  def runnerTask: Initialize[Task[ScalaRun]] = runnerInit
+  def runnerTask: Setting[Task[ScalaRun]] = runner := runnerInit.value
```

Comment:
But the only motivation for this change is to restore source-compatibility with 0.13..